File: Overriding.java
class Student{
    void display()
    {
        System.out.println("I am a student");
    }
}
class MVGRStudent extends Student{
    void display()
    {
        System.out.println("I am a student of MVGR");
    }
}
class Main{
    public static void main(String[] args) {
        MVGRStudent obj=new MVGRStudent();
        obj.display();
    }
}

File: Overriding.cpp
#include<iostream>
using namespace std;
class Student
{
    public:
        void display()
        {
            cout<<"I am a student"<<endl;
        }
};
class MVGRStudent : Student{
    public:
        void display()
        {
            cout<<"I am a student of MVGR"<<endl;
        }
};
int main()
{
    MVGRStudent obj;
    obj.display();
}
